redis/src/caching/statistics.rs

```diff
@@ -0,0 +1,49 @@
+use std::sync::atomic::{AtomicUsize, Ordering};
+use std::sync::Arc;
+
+/// CacheStatistics holds statistics generated by Client Side Caching.
+#[derive(Debug)]
+pub struct CacheStatistics {
```

Comment:
Maybe `FinalizedStatistics`? I want a name that clarifies this was computed from `Statistics` and is its final form.

Reply:
I chose `CacheStatistics` because `Statistics` isn't a public struct and users don't have to know about it. So `FinalizedStatistics` might bring more confusion.